src/EA.Weee.Web/Areas/Producer/Views/Producer/ViewOrganisation/ContactDetails.cshtml

```diff
@@ -15,15 +15,71 @@
         @Html.Partial("ViewOrganisation/_OrganisationDetailsTabs", Model)
 
         <div class="govuk-tabs__panel" id="organisationDetails">
-            <h1 class="govuk-heading-m">
+            <h2 class="govuk-heading-m">
                 @title
-            </h1>
-            <table id="pcs-details-table" class="govuk-table">
+            </h2>
+            <table id="pcs-contact-details-table" class="govuk-table">
                 <tbody class="govuk-table__body">
-                   
+                    <tr class="govuk-table__row">
+                        <th class="govuk-table__header" scope="row">First Name</th>
+                        <td class="govuk-table__cell">@Model.SmallProducerSubmissionData.ContactData.FirstName</td>
+                    </tr>
+
+                    <tr class="govuk-table__row">
+                        <th class="govuk-table__header" scope="row">Last name</th>
+                        <td class="govuk-table__cell">@Model.SmallProducerSubmissionData.ContactData.LastName</td>
+                    </tr>
+
+                    <tr class="govuk-table__row">
+                        <th class="govuk-table__header" scope="row">Position</th>
+                        <td class="govuk-table__cell">@Model.SmallProducerSubmissionData.ContactData.Position</td>
+                    </tr>
+
+                    <tr class="govuk-table__row">
+                        <th class="govuk-table__header" scope="row">Email</th>
+                        <td class="govuk-table__cell">@Model.SmallProducerSubmissionData.CurrentSubmission.ContactAddressData.Email</td>
```

Comment:
@stevelee-civica  looking at this it looks like we also need to have the Contact address data returned from GetSmallProducerSubmissionHandler. So add an AddressData property to SmallProducerSubmissionData and map that in 
![image](https://github.com/user-attachments/assets/c7df5ff2-dc67-4ef4-98f5-fa9996524a8f)

Using the mapper.Map<Address, AddressData>(directRegistrant.Address) mapper similar to how the contact data is done and use that to display these fields.

Reply:
@stevelee-civica  and update the unit tests :)

Reply:
updated